### DISCUSSION TITLE
CMOR-ing PISM output

### DISCUSSION THREAD

**Discussion by aaschwanden** [pism/pism#500] [Ideas]
Constantine, for ISMIP6 we're expecting to write CMOR compliant files:

http://www2-pcmdi.llnl.gov/cmor

Can we look into how we can make PISM output compatible?

**Comment by ckhroulev** [comment]
I could not find a document describing what "CMOR compliance" means. Could you point me in the right direction? (An automated test similar to the `cfchecker` would be great.)

**Comment by ckhroulev** [comment]
A better link: http://cmor.llnl.gov/
